tools/clang/test/CodeGenSPIRV/spv.enable-maximal-reconvergence.hlsl

```diff
@@ -0,0 +1,8 @@
+// RUN: %dxc -T cs_6_0 -E main -fspv-enable-maximal-reconvergence %s -spirv | FileCheck %s
```

Comment:
Could you add another run command that checks that the extension and execution mode are not there when the option is not used?

Reply:
Sure, done

Reply:
You don't need a separate file. You can add multiple run commands on one file with different patterns to look for.
```suggestion
// RUN: %dxc -T cs_6_0 -E main -fspv-enable-maximal-reconvergence %s -spirv | FileCheck %s -check-prefix=ENABLED
// RUN: %dxc -T cs_6_0 -E main %s -spirv | FileCheck %s -check-prefix=DISABLED
```

Then you can add the checks for using `ENABLED` or `DISABLED` instead of `CHECK`.

Reply:
Ah yes, better to keep them close. Merged the 2 tests.